Content.Server/GameObjects/Components/GUI/InventoryComponent.cs

```diff
@@ -36,6 +36,8 @@ public class InventoryComponent : SharedInventoryComponent, IExAct, IEffectBlock
         [ViewVariables]
         private readonly Dictionary<Slots, ContainerSlot> SlotContainers = new Dictionary<Slots, ContainerSlot>();
 
+        private KeyValuePair<Slots, Tuple<EntityUid, bool>>? HoverEntity;
```

Comment:
Use `ValueTuple` instead of `Tuple` instead here. (`private KeyValuePair<Slots, (EntityUid entity, bool fits)>? HoverEntity;`)

Reply:
done